deploy/olm-catalog/3scale-operator-master/0.0.0/3scale-operator-master.v0.0.0.clusterserviceversion.yaml

```diff
@@ -103,6 +103,48 @@ metadata:
             },
             "username": "admin"
           }
+        },
+        {
+          "apiVersion": "capabilities.3scale.net/v1beta1",
+          "kind": "Account",
+          "metadata": {
+            "name": "example-account"
+          },
+          "spec": {
+            "size": 3
```

Comment:
I guess at this point in this PR this is expected because we are waiting to have all changes applied before regenerating the CSV. Just let's not forget about updating the CRDs files and this correctly, as right now it's a generic CRD value (spec.size)